Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.12-alpine
+
+WORKDIR /code
+
+COPY ./requirements.txt /code/requirements.txt
+
+RUN pip install --no-cache-dir --upgrade -r /code/requirements.txt
+
+COPY ./src /code/src
+
+CMD ["gunicorn", "src.main:app", "-w", "1", "-k", "uvicorn.workers.UvicornWorker", "--bind", "0.0.0.0:1234", "--log-syslog", "--log-level", "info"]
```

Comment:
if you bind on `1234`, don't you need to add a `PORT` directive?

Reply:
As pointed out in Discord DMs - EXPOSE, and yee I forgot it. Will be added in upcoming commit